cmds/core/gzip/gzip.go

```diff
@@ -79,4 +68,30 @@ func main() {
 			continue
 		}
 	}
+
+	return nil
+}
+
+func main() {
+	var opts gzip.Options
+	cmdLine.Usage = usage
+
+	if err := opts.ParseArgs(os.Args, cmdLine); err != nil {
+		if errors.Is(err, gzip.ErrStdoutNoForce) {
+			fmt.Fprintf(os.Stderr, "%s\n", err)
+			os.Exit(1)
+		}
+		if errors.Is(err, gzip.ErrHelp) {
+			cmdLine.Usage()
+			os.Exit(0)
```

Comment:
if it is a usage error, should we exit with 1? This is an error, right?

Reply:
This is intentional, both linux and macOs gzip return 0 if help was requested. Also package logic used empty error to signal that help is requested and it's confusing. So this is not a real error. I'm not a fan of such approach, but we can live with that I think.